### PR TITLE
[Bugfix] Fix buy tab: Simplex urls

### PR DESCRIPTION
**Pull request by kcw-grunt**
## Problem
The LW backend has been updated with a new url and service and the vales do not currently work when the user will try to sort out the issue.

## Approach 
Run tests for the new service by starting with staging/sandbox
- Run end to end test
- Add in the constants for production

## Video Clips
|Completed Transaction|
|-----------------------|
|sandbaox|

### CODE REVIEW

File: loafwallet/BRAPIClient+Wallet.swift [hunk 1]
```diff
@@ -7,8 +7,8 @@
 
 import Foundation
 
-private let ratesURL = "https://api.loshan.co.uk/api/v1/rates"
-private let fallbackRatesURL = "https://api.loafwallet.org/api/v1/rates"
+private let ratesURL = "https://api-prod.lite-wallet.org/rates"
+private let fallbackRatesURL = "https://api-stage.lite-wallet.org/rates"
```
Comment (mosadialiou) [inline loafwallet/BRAPIClient+Wallet.swift:11]: can you share here the code where `ratesURL` and `fallbackRatesURL` are used?
From my understanding I would have just write something like this:
    
#if DEBUG
    static let ratesURL = "https://api-stage.lite-wallet.org/rates"
    #else
    static let ratesURL = "https://api-prod.lite-wallet.org/rates"
    #endif

Thoughts?

File: loafwallet/partner-keys.plist [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mosadialiou) [inline loafwallet/partner-keys.plist:1]: to be removed from git and committed files

**Review by mosadialiou — CHANGES_REQUESTED** [review]

File: loafwallet/BRAPIClient+Wallet.swift (hunk 1 above)
Comment (kcw-grunt) [reply loafwallet/BRAPIClient+Wallet.swift:11]: Good point.  There should be served by the API server.
I substituted the url and expected it to fail
@gneo54 is this endpoint _(https://api-stage.lite-wallet.org/v1/rates)_ ready?

Comment (kcw-grunt) [reply loafwallet/BRAPIClient+Wallet.swift:11]: @mosadialiou we can just do this:
APIServer.baseUrl + "v1/rates"
After the confirmation

Comment (mosadialiou) [reply loafwallet/BRAPIClient+Wallet.swift]: Yes sure!

**Comment by kcw-grunt** [comment]
@gneo54
@mosadialiou :

Ran another sandbox run and was successfully completed.

|Sandbox-Run: 2|
|----------------|
|sandbox-run-2|

File: loafwallet/BRAPIClient+Wallet.swift [hunk 3]
```diff
@@ -22,7 +21,7 @@ extension BRAPIClient {
     }
     
     func exchangeRates(isFallback: Bool = false, _ handler: @escaping (_ rates: [Rate], _ error: String?) -> Void) {
-        let request = isFallback ? URLRequest(url: URL(string: fallbackRatesURL)!) : URLRequest(url: URL(string: ratesURL)!)
+        let request = isFallback ? URLRequest(url: URL(string: fallbackRatesURL)!) : URLRequest(url: URL(string: APIServer.baseUrl + "v1/rates")!)
```
Comment (mosadialiou) [inline loafwallet/BRAPIClient+Wallet.swift:24]: When we get a chance, just write this:
let request = URLRequest(url: URL(string: isFallback ? fallbackRatesURL : APIServer.baseUrl + "v1/rates")!)

**Pull request merged**